pkg/ibmvpc/examples/out/explain_out/tgwDisablesTCPRespond_all_vpcs_explain_debug.txt

```diff
@@ -0,0 +1,44 @@
+Explaining connectivity from ky-vsi0-subnet0 to ky-vsi0-subnet10
+================================================================
+
+Allowed connections from test-vpc0-ky/ky-vsi0-subnet0[10.240.0.5] to test-vpc1-ky/ky-vsi0-subnet10[10.240.64.4]: All Connections
+	TCP response is blocked
+
+Path:
+	ky-vsi0-subnet0[10.240.0.5] -> security group sg1-ky -> subnet0 -> network ACL acl1-ky -> 
+	test-vpc0-ky -> TGW local-tg-ky -> test-vpc1-ky -> 
+	network ACL acl11-ky -> subnet10 -> security group sg11-ky -> ky-vsi0-subnet10[10.240.64.4]
+
+
+Details:
+~~~~~~~~
+Path is enabled by the following rules:
+	Egress:
+		security group sg1-ky allows connection with the following allow rules
+			index: 1, direction: outbound,  conns: protocol: all, remote: 0.0.0.0/0, local: 0.0.0.0/0
+		network ACL acl1-ky allows connection with the following allow rules
+			index: 0, direction: outbound , src: 0.0.0.0/0 , dst: 0.0.0.0/0, conn: all, action: allow
+
+	transit gateway local-tg-ky allows connection via transit connection tg_connection1 with the following prefix filter
+		default prefix,  action: permit
+
+	Ingress:
+		network ACL acl11-ky allows connection with the following allow rules
+			index: 2, direction: inbound , src: 0.0.0.0/0 , dst: 0.0.0.0/0, conn: all, action: allow
+		security group sg11-ky allows connection with the following allow rules
+			index: 1, direction: inbound,  conns: protocol: all, remote: 0.0.0.0/0, local: 0.0.0.0/0
+
+TCP response is disabled by the following rules:
+	Egress:
+		network ACL acl11-ky allows connection with the following allow rules
+			index: 1, direction: outbound , src: 0.0.0.0/0 , dst: 0.0.0.0/0, conn: all, action: allow
```

Comment:
Lets have a single header. Otherwise it will get much too messy, especially given that the enabling and disabling rules may be mixed.  
Maybe: `Relevant rules to disabled path?` 
(The enabling rules can stay as it is now since for a path to be enabled all rules must enable it)

Reply:
Path is disabled (enabled); the relevant rules are

Reply:
done